assets/js/modules/search-console/dashboard-details/dashboard-details-widget-keyword-table.js

```diff
@@ -25,13 +25,17 @@ const { __, sprintf } = wp.i18n;
 class DashboardDetailsWidgetKeywordsTable extends Component {
 
 	render() {
+		const context = googlesitekit.permaLink ? __( 'page', 'google-site-kit' ) : __( 'site', 'google-site-kit' );
+
+		/* translators: %s: Context of the top search queries, page or site. */
+		const title = sprintf( __( 'Top search queries for your %s', 'google-site-kit' ), context );
```

Comment:
That's not really friendly for translators. It's preferable to have full sentences. In some languages the pronouns change depending on the noun. Correctly translating the sentence here is not possible when using placeholders.

So this should rather use something like this:

```js
const title = googlesitekit.permaLink ? sprintf( __( 'Top search queries for your page', 'google-site-kit' ), context ) : __( 'Top search queries for your site', 'google-site-kit' );
```

Reply:
@swissspidy thanks for the feedback, I have pushed the changes.